### ISSUE TITLE
Create emails with User-Agent header

### ISSUE THREAD

**Issue by chibenwa** [linagora/tmail-flutter#221]
Email can have the `User-Agent` header field, that tells where they were sent from.

Eg:

```
User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:78.0) Gecko/20100101
 Thunderbird/78.13.0
```

It would be nice to specify the user agent... You can add the following property to the `Email/set create` email...

```
"header:User-Agent:asText": "Android/XXX TeamMail/1.0"
```

**Comment by dab246** [comment]
@chibenwa  Please help me with this problem.

Why is `Email/set` disallowed of the `headers` attribute in `Email`? Although in the [jmap.io](https://jmap.io/spec-mail.html) documentation there is a description of that property. 

<img width="1439" alt="Screen Shot 2022-02-14 at 17 32 04" src="https://user-images.githubusercontent.com/80730648/153847709-d5f5cc9a-ed54-47f9-aa7b-00ab0400991e.png">

I have tried adding `header` to `Email/set create` but always get this result.

https://user-images.githubusercontent.com/80730648/153847740-c81fdf5f-b0ee-4f46-923e-c519a11294d9.mov

**Comment by chibenwa** [comment]
You can set individual headers, eg `"header:User_Agent:asText": "Android/XXX TeamMail/1.0"` as stated above in ticket description.

TLDR dont set the all headers only set individual headers.
